### ISSUE TITLE
Trip computer

### ISSUE THREAD

**Issue by Teyar** [EDDiscovery/EDDiscovery#1954]
Once again the question about the reset of the travel distance in the trip computer when the Start button is pressed.
(I refer to the ticket #1942)

If I have understood correctly, a starting point should be set in the journal when the Start button is pressed and the travel distance in the trip computer will be reset at the next journal entry.
(I may not have understood it correctly. My English is not good and I have to work with a translation program. And what comes out sometimes, you sure know yourself too well.)

However, the problem with the reset is still there. The starting point in the journal is set, but the travel distance in the trip computer is not reset.

I don't know if it helps anything, but appendix are the log files since my last ticket.
[Log.zip](https://github.com/EDDiscovery/EDDiscovery/files/2009006/Log.zip)

Thank you for your efforts

**Comment by klightspeed** [comment]
It looks like the Start button on the trip panel is supposed to put a stop marker on the preceding FSD Jump in order to reset the counters.  It sounds like this is not happening.

This is due to a typo at https://github.com/EDDiscovery/EDDiscovery/blame/master/EDDiscovery/UserControls/UserControlTrippanel.cs#L283

You can work around this until it is fixed by selecting the entry immediately before the start entry, right-click, and click `Set start/stop point for travel calculations`.

**Comment by robbyxp1** [comment]
Thanks

**Comment by Teyar** [comment]
Hello again

I just did it the way you wrote and added the start/Stop point in the journal about right-click....
I noticed that a red flag is set and also the travel distance is reset in the trip computer. I think that's the stop-point.
When I press the Start button on the Trip computer (currently) only one green flag is set, which probably indicates the starting point. Therefore, the trip computer will not be reset either.
I hadn't noticed that before.

You already suspected it.
I just wanted to get the feedback.

(I think I'll probably be more careful about what happens when I receive later error messages or feedback.)

**Comment by Teyar** [comment]
Well, the trip computer was reset and I flew.
In the next system no travel distance in the trip computer. Was probably at the missing starting point. So I added the starting point (again via right click... in the journal).

Then I got the following message:

**An unhandled fatal exception has occurred.

= = = = Begin = = = =
Newton Soft. Json. JsonReaderException: Unexpected character encountered while parsing value: <. Path ' ', line 0, position 0.
   by Newton Soft. Json. JsonTextReader. Parsevalue ()
   At Newton Soft. Json. Linq. JObject. Load (JsonReader reader, JsonLoadSettings settings)
   by Newton Soft. JSON. Linq. JObject. Parse (string json, JsonLoadSettings settings)
   At EliteDangerousCore. EDSM. EDSMClass. Getcomments (Action ' 1 logout) in C:\Code\EDDiscovery\EliteDangerous\EDSM\EDSMClass.cs: Line 302.
   At EliteDangerousCore. EDSM. EDSMLogFetcher. FetcherThreadProc () in C:\Code\EDDiscovery\EliteDangerous\EDSM\EDSMLogFetcher.cs: Line 94.
   at System. Threading. ExecutionContext. RunInternal (ExecutionContext ExecutionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   at System. Threading. ExecutionContext. Run (ExecutionContext ExecutionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   at System. Threading. ExecutionContext. Run (ExecutionContext ExecutionContext, ContextCallback callback, object state)
   at System. Threading. ThreadHelper. Thread Start ()
= = = = = End = = = = =**

When closing the error message, also includes EDD.

After I finished the game, I started EDD again. Again came the same error message and in the Log window is the following:

Checking for new EDDiscovery maps
Checking for new Expedition data
Loaded Notes, Bookmarks and Galactic mapping.
Reading travel history
Scanning for bmp screenshots in C:\Users\thgen\OneDrive\Bilder\Frontier Developments\Elite Dangerous
Checking for updated EDSM systems (may take a few moments).
Downloading systems from UTC 17.05.2018 07:50:10 to 17.05.2018 08:51:28
Logged into Companion API
**Check Systems exception: Unexpected character encountered while parsing value: <. Path '', line 0, position 0.
Trace:    bei Newtonsoft.Json.JsonTextReader.ParseValue()
   bei EliteDangerousCore.EDSM.SystemClassEDSM.DoParseEDSMUpdateSystemsReader(JsonTextReader jr, Boolean[] grididallowed, DateTime& maxdate, Func`1 cancelRequested, Action`2 reportProgress, Boolean useCache, Boolean useTempSystems) in C:\Code\EDDiscovery\EliteDangerous\EDSM\SystemClassEDSM.cs:Zeile 295.
   bei EliteDangerousCore.EDSM.SystemClassEDSM.ParseEDSMUpdateSystemsStream(TextReader sr, Boolean[] grididallow, DateTime& date, Boolean removenonedsmids, Func`1 cancelRequested, Action`2 reportProgress, Boolean useCache, Boolean useTempSystems) in C:\Code\EDDiscovery\EliteDangerous\EDSM\SystemClassEDSM.cs:Zeile 265.
   bei EliteDangerousCore.EDSM.SystemClassEDSM.ParseEDSMUpdateSystemsString(String json, Boolean[] grididallow, DateTime& date, Boolean removenonedsmids, Func`1 cancelRequested, Action`2 reportProgress, Boolean useCache) in C:\Code\EDDiscovery\EliteDangerous\EDSM\SystemClassEDSM.cs:Zeile 254.
   bei EliteDangerousCore.EDSM.SystemClassEDSM.PerformEDSMUpdateSync(Boolean[] grididallow, Func`1 PendingClose, Action`2 ReportProgress, Action`1 LogLine, Action`1 LogLineHighlight) in C:\Code\EDDiscovery\EliteDangerous\EDSM\SystemClassEDSM.cs:Zeile 744.
   bei EDDiscovery.EDDiscoveryController.DoPerformSync() in C:\Code\EDDiscovery\EDDiscovery\EDDiscoveryController.cs:Zeile 696.**
History refresh complete.
Loading completed, total of 22.454.327 systems stored

Since then I can no longer use EDD since every time after the start the error message comes up and if I close it also EDD closes or crashes.

I have attached the log files again.
[Log_17052018.zip](https://github.com/EDDiscovery/EDDiscovery/files/2012457/Log_17052018.zip)
